glossary.yml

```
@@ -4136,6 +4141,10 @@
     term: "módulo"
     def: >
       un paquete de software reutilizable, también se llama una [biblioteca](#library).
+  xh:
+    term: "Imodyuli"
+    def: >
+      Iphakheji yesoftware](#package) enokuphinda isebenze, ekwabizwa ngokuba [yilayibrari](#library)
```

I believe this is where the link should go? 

```suggestion
      [Iphakheji yesoftware](#package) enokuphinda isebenze, ekwabizwa ngokuba [yilayibrari](#library)
```

---

@zkamvar indeed!